tests/controllers/test_collection_instrument.py

```diff
@@ -64,25 +61,14 @@ def test_get_instrument_by_search_string_invalid_search_value(self):
         with self.assertRaises(RasDatabaseError):
             self.collection_instrument.get_instrument_by_search_string('{\"COLLECTION_EXERCISE\": \"invalid_uuid\"}')
 
-    def test_get_instrument_by_id(self):
```

Comment:
Do we need a test for get_instrument_json_by_id?

Reply:
The view test covers it